oprf/group/group.go

```diff
@@ -0,0 +1,206 @@
+package group
+
+import (
+	"crypto/subtle"
+	"errors"
+	"math/big"
+)
+
+// Element is a representation of a group element. It has two coordinates of
+// *big.Int type.
+type Element struct {
+	c   *Ciphersuite
+	x   *big.Int
+	y   *big.Int
+	com bool // use compression when serialiazing, be default set to true
```

Comment:
Where do we ever set this to `false`? Also, shouldn't it be a property of the ciphersuite? For example, [the draft](https://cfrg.github.io/draft-irtf-cfrg-voprf/draft-irtf-cfrg-voprf.html#name-oprfp-384-sha-512 specifies that serialized points for certain ciphersuites are compressed.

Reply:
The idea is to move the group functionality eventually outside (in a subsequent PR), so it can be reused for different things besides OPRF.

Reply:
To see when it is defined if it is compressed or not, there is the `isCompressed` function.

Reply:
and also on every call to `NewElement` is set to true.

Reply:
Sorry, my point is that if the value is never set to `false`, why make it a parameter at all?

Reply:
Because the group functionality will eventually live outside the OPRF to be used be functionality that might need it to be set false.

Reply:
That's fine, but it's *not used today*, and I don't think we should be adding code that's not used or required for a specific purpose.

Reply:
Sure. I'll remove.